components/PageNavigation.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <b-navbar
+    class="navigation-container border-bottom"
+    data-qa="main navigation"
+  >
+    <b-container class="p-0">
+      <b-navbar-nav class="px-2">
+        <li
+          v-for="nav in navigation"
+          :key="nav.text"
+          class="nav-item"
+        >
+          <SmartLink
+            :destination="localePath({name: 'slug', params: { pathMatch: nav.url.replace(/^\//, '') }})"
```

Comment:
this should just take the url. The localepath call and the '/' replacement should only happen in the smart link component for the case of the destination not being a url.
Now this will break where a top nav link goes to an external site (see the updated 'blogs' one). Also we'll be duplicating this logic for other smart links if we don't move it into the component.

Reply:
How do we approach this logic? Do we pass a property to let it know we need to add a locale path?

Reply:
just always use a locale path. I can't think of a scenario where we'd want to link to something not in the selected locale.